content/en/_index.md

```diff
@@ -961,6 +1016,8 @@ one notable exception:
 
 ### Portability {#portability}
 
+**Applies to:** ✂︎ Snippets, 🧩 Patterns
```

Comment:
This may not apply to patterns at all (maybe just best effort). There are several sample apps that I've seen that are about doing something specifically in Windows, etc.

Reply:
Added a note that exceptions are for samples showcasing platform-specific functionality.